scripts/jams_to_mirex_pattern.py

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
There should be a shebang and docstring comment explaining what this does and how to use it:
```
#!/usr/bin/env python
'''Convert JAMS to MIREX pattern discovery format

Example usage:

./jams_to_mirex_pattern.py /path/to/jams.py /path/to/mirex_output.csv
'''

or something to that effect.